Backend/app.js

```diff
@@ -13,12 +13,19 @@ import knex from "./config/knex.js";
 
 const app = express();
 app.use(morgan("dev"));
-app.use(cors());
+const corsOptions = {
+  origin: 'http://localhost:3000',
```

Comment:
use environment variables to access because value will change with different Environments.

Reply:
added proper Add .env file for environment variables.you can see the updates in the latest commit.